numpy/core/tests/examples/cython/setup.py

```diff
@@ -1,29 +0,0 @@
-"""
```

Comment:
Here we don't use `numpy.distutils`, so this should continue to work unchanged. Hence it may be useful to keep around as an example? No need to test it for me, because it's so simple that it should be robust.

Although if you prefer to clean it up, also okay - no strong feelings about it. For the `numpy/random` I didn't think about this, but also that used `numpy.distutils` I think, so needed to go anyway.